src/js/components/Calendar/Calendar.js

```diff
@@ -630,10 +630,23 @@ const Calendar = forwardRef(
           </Box>
           <Box flex={false} direction="row" align="center">
             <Button
-              a11yTitle={previousMonth.toLocaleDateString(locale, {
-                month: 'long',
-                year: 'numeric',
-              })}
+              a11yTitle={
+                messages?.previous
```

Comment:
this shouldn't be checking messages passed it. It should be letting format figure it out. Ie. if messages wasn't passed in, it should still be getting it from the global messages.

Reply:
So, that is why I check if messages.previous was passed.
If the Calendar is rendered without messages it will use the default message in the else condition: `previousMonth.toLocaleDateString(locale, {
                      month: 'long',
                      year: 'numeric',
                    })`  which is the current master behavior.
If I set messages.next and messages.previous the aria-label keeps in the default e.g.: "Moved to November 2021", so when we set the prop it will use the messages as aria-label

Reply:
Is there a reason you don't want the default to have the 'Moved to' text? That seems to be a better default than what it used to be.

Reply:
I'm not sure if I get yor point.
If you don't set the messages prop it will keep the 'Moved to'
If you set the messages the aria-label will be what you set. 
So when you turn on voice over and focus in the chevron you will heard the same message when you click.
The current master changes the onclick event but not the focus.

Reply:
> If you don't set the messages prop it will keep the 'Moved to'

Only for the announce during onClick. The aria-label will still be just the date.

> If you set the messages the aria-label will be what you set. So when you turn on voice over and focus in the chevron you will heard the same message when you click. 

Which is great but I'd think that's also what we want when the messages prop isn't set so apps which aren't passing messages to Calendar itself get the better experience.

Reply:
If I'm following this correctly, I think we can just do the following:
```
a11yTitle={
   format({
       id: 'calendar.previous',
        messages,
        values: {
           date: previousMonth.toLocaleDateString(locale, {
               month: 'long',
               year: 'numeric',
           }),
        },
    })
}
```

`format` function will be able to determine if the caller has passed a custom `message` or if we should rely on the default.

Reply:
It worked! Thank you all 😀